src/Components/CreateImageWizard/CreateImageWizard.tsx

```diff
@@ -113,6 +113,7 @@ export const CustomWizardFooter = ({
   const { goToNextStep, goToPrevStep, goToStepById, close, activeStep } =
     useWizardContext();
   const { analytics } = useChrome();
+  const isOnPremise = useIsOnPremise();
```

Comment:
And I think the `isOnPremise` here could be passed down from `CreateImageWizard`.